vendor/engines/split_accounts/app/models/concerns/account_extension.rb

```diff
@@ -0,0 +1,11 @@
+module Concerns
```

Comment:
I don't think that it will in this case, but I'm not sure. Even if it's within a `Concerns` namespace, it should probably still be in the `SplitAccounts` namespace.